Utilities/Install-CustomScript.ps1

```diff
@@ -0,0 +1,116 @@
+##############################################################################################
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+#
+# Install-CustomScript.ps1
+<#
+.SYNOPSIS
+
+.PARAMETER
+	-AzureSecretsFile, the path of Azure secrets file
+	-VmName, the name of the VM to install the CustomScript extension, if no VmName is provided, it will be installed on all VMs
+	-ResourceGroupName, the resource group name of the VMs to install the CustomScript extension, if no ResourceGroupName is provided, it will be installed on all VMs in the subscription
+	-FileUris, comma separated Uris of the custom scripts
+	-CommandToRun, the command to run on the VM
+	-StorageAccountName, the name of the storage account that contains the custom scripts
+	-StorageAccountKey, the key of the storage account that contains the custom scripts
+	-OSType, the type of the OS, Linux or Windows, valid only if neither VmName nor ResourceGroupName is provided
+
+.NOTES
+	Creation Date:
+	Purpose/Change:
+
+.EXAMPLE
+	.\Utilities\Install-CustomScript.ps1 -AzureSecretsFile <PathToSecretFile> -VmName <VmName> -ResourceGroupName <RgName> `
+		-FileUris "https://teststorageaccout.blob.core.windows.net/script/install.sh" -CommandToRun "bash install.sh" `
```

Comment:
scripts should not reside on a location without versioning.
That is bad for code maintenance and tracking of changes.

Reply:
agree, this is for installing security packages on all VMs in our subscription, to meet the security coverage, the installation script is put in Azure storage as it contains internal packages, I will add version to the internal script